galsim/interpolatedimage.py

```diff
@@ -638,24 +679,24 @@ def __init__(self, real_kimage, imag_kimage, k_interpolant=None, stepk=None,
             raise ValueError("Real and Imag kimages must have same scale.")
 
         # Make sure any `wcs`s are `PixelScale`s.
-        if ((real_kimage.wcs is not None
-             and not real_kimage.wcs.isPixelScale())
-            or (imag_kimage.wcs is not None
-                and not imag_kimage.wcs.isPixelScale())):
+        if ((real_kimage.wcs is not None and
+             not real_kimage.wcs.isPixelScale()) or
+            (imag_kimage.wcs is not None and
+                not imag_kimage.wcs.isPixelScale())):
             raise ValueError("Real and Imag kimage wcs's must be PixelScale's or None.")
 
         # Check for Hermitian symmetry properties of real_kimage and imag_kimage
         shape = real_kimage.array.shape
         # If image is even-sized, ignore first row/column since in this case not every pixel has
         # a symmetric partner to which to compare.
-        bd = galsim.BoundsI(real_kimage.xmin + (1 if shape[1]%2==0 else 0),
+        bd = galsim.BoundsI(real_kimage.xmin + (1 if shape[1] % 2 == 0 else 0),
                             real_kimage.xmax,
-                            real_kimage.ymin + (1 if shape[0]%2==0 else 0),
+                            real_kimage.ymin + (1 if shape[0] % 2 == 0 else 0),
```

Comment:
Here's another case where the original was better.  Leaving out the spaces in `shape[0]%2==0` helps the reader easily see what's going on with the `if`/`else`.  Adding in spaces actually makes harder to parse the logical parts of this line.  Again, I wish you could just trust the style sensibilities of the original coder rather than some black box linter that completely ignores the first rule in pep8: "A Foolish Consistency is the Hobgoblin of Little Minds".  

Flake8 doesn't know what to do with that rule, so it just skips it.  But it's actually the most important rule in the whole thing.  And it's why I can't stand style linters.  Code linters (like pyflake) are fine and helpful.  But flake8 is not as far as I'm concerned.  I haven't seen a single style change you made here to satisfy flake8 that makes anything clearer, and I've seen lots that make things just a little bit worse.  I haven't commented on most of them, because they just aren't a big enough deal for me to bother, but there are at least a dozen or two more that I think are better the way they were for reasons similar to this one.